router/router.go

```diff
@@ -155,7 +158,12 @@ func (rt *HandleT) workerProcess(worker *workerT) {
 			logger.Debugf("[%v Router] :: trying to send payload %v of %v", rt.destID, attempts, ser)
 
 			deliveryTimeStat.Start()
-			respStatusCode, respStatus, respBody = rt.netHandle.sendPost(job.EventPayload)
+			if misc.ContainsString(objectStreamDestination, job.CustomVal) {
+				respStatusCode, respStatus, respBody = streammanager.Produce(job.EventPayload, job.CustomVal)
```

Comment:
Please check if there are any pending connections after the test. Everything should be closed.

Reply:
This need not be specific to streamDestinations. We should use this for all destinations that don't have a default HTTP endpoint, but have their own SDKs that enable us to send events.

I can't think of a good name but something like
customDestinationManager.send(job.EventPayload, job.CustomVal)

Reply:
incorporated.